src/module/iotjs_module_i2c.c

```diff
@@ -0,0 +1,418 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "iotjs_def.h"
+#include "iotjs_objectwrap.h"
+#include "iotjs_module_i2c.h"
+
+
+#define THIS iotjs_i2creqwrap_t* i2creqwrap
+void iotjs_i2creqwrap_initialize(THIS, const iotjs_jval_t* jcallback,
```

Comment:
Please insert one or two blank lines between ``define`` and function declaration.